src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -1508,6 +1520,11 @@ class EltwiseRefExecutor : public Eltwise::IEltwiseExecutor {
                     case Algorithm::EltwisePrelu:             *dst_ptr_f = src_f[0] > 0 ? src_f[0] : src_f[0] * src_f[1]; break;
                     case Algorithm::EltwiseErf:               *dst_ptr_f = std::erf(src_f[0]); break;
                     case Algorithm::EltwiseSoftSign:          *dst_ptr_f = src_f[0] / (1 + std::fabs(src_f[0])); break;
+                    case Algorithm::EltwiseIsFinite:          *dst_ptr_f = std::isfinite(src_f[0]); break;
+                    case Algorithm::EltwiseIsInf:
+                        *dst_ptr_f = std::isinf(src_f[0]) && (_opData.detectNegative && (src_f[0] < 0) || _opData.detectPositive && (src_f[0] > 0));
```

Comment:
Can you just compare with `std::numeric_limits<float>::infinity()`  and `-std::numeric_limits<float>::infinity();` to avoid additional check with zero?

Reply:
Done